universal-login-sdk/lib/api/sdk.ts

```diff
@@ -123,20 +123,38 @@ class UniversalLoginSDK {
     );
   }
 
+  removeGasLimit(message: Partial<Message>) {
```

Comment:
`withoutGasLimit`

Reply:
this is actually obsolete. Will remove this